setup.py

```diff
@@ -16,24 +16,31 @@
     name="orion.algo.HEBO",
     version=versioneer.get_version(),
     cmdclass=versioneer.get_cmdclass(),
-    description="TODO",
+    description="Orion adapter wrapper for the HEBO algorithm.",
     long_description=open(
         os.path.join(repo_root, "README.rst"), "rt", encoding="utf8"
     ).read(),
     license="BSD-3-Clause",
     author="Epistimio",
-    author_email="xavier.bouthillier@mila.quebec",
+    author_email="normandf@mila.quebec",
     url="https://github.com/Epistimio/orion.algo.HEBO",
     packages=["orion.algo.HEBO"],
     package_dir={"": "src"},
     include_package_data=True,
     entry_points={
-        "BaseAlgorithm": ["HEBO_HEBO = orion.algo.HEBO.hebo:HEBO"],
+        "BaseAlgorithm": ["HEBO_HEBO = orion.algo.HEBO.hebo_algo:HEBO"],
     },
-    install_requires=["orion>=0.1.15", "numpy"],
+    install_requires=[
+        "orion>=0.1.15",
```

Comment:
```suggestion
        "orion>=0.2.2",
```
Oops, i'll need to fix this in every plugins :sweat_smile:

Reply:
Fixed in 900677887bb43e5208da4c4e393cb25a251a3b69